api/v1/mongodbcommunity_types.go

```diff
@@ -101,6 +101,10 @@ type MongoDBCommunitySpec struct {
 	// +kubebuilder:pruning:PreserveUnknownFields
 	// +nullable
 	AdditionalMongodConfig MongodConfiguration `json:"additionalMongodConfig,omitempty"`
+
+	// AutomationConfigOverride is merged on top of the operator created automation config. Processes are merged
+	// by name.
+	AutomationConfigOverride *AutomationConfigOverride `json:"automationConfigOverride,omitempty"`
```

Comment:
Should we indicate here that only `process.Disabled` is supported right now?

Reply:
added comment